data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/damage/damage_multiply_fraction.cfg

```diff
@@ -26,8 +26,42 @@
             [/effect]
         [/modify_unit]
 
-        {ATTACK_AND_VALIDATE 6 (DAMAGE_VALUE=) WEAPON_COUNT=2}
+        {ATTACK_AND_VALIDATE 8 (DAMAGE_VALUE=) WEAPON_COUNT=2}
         {SUCCEED}
     [/event]
 ) SIDE2_LEADER="Elvish Archer"}
 #endif
+
+#####
+# API(s) being tested: [damage]value=
+##
+# Actions:
+# Give the leaders a damage ability with  multiply attribute
+# Have the side 1 leader attack the side 2 leader with both its weapons
+##
+# Expected end state:
+# The side 1 leader has 2 weapons both of which have 8 damage (4*1.95=7.8 rounded up to 8)
```

Comment:
According to https://github.com/wesnoth/wesnoth/issues/9467#issuecomment-2416966204 rounding has to be towards original value. 4*1.95=7.8, rounding is towards original value 4, so expected damage is 7.

Reply:
That would match what I remember of damage calculation. But maybe it was only meant like 7.5 original 4 = 7, while 7.5 original 40 = 8

Reply:
In any case, either test is wrong, or test does not cover what it should cover. It is especially important to test edge case, such as `4*0.875 and 4*1.875` to see how .5 value sometimes goes up, sometimes down.

Reply:
> According to [#9467 (comment)](https://github.com/wesnoth/wesnoth/issues/9467#issuecomment-2416966204) rounding has to be towards original value. 4*1.95=7.8, rounding is towards original value 4, so expected damage is 7.

No no and no, before value truncated of 7.8 to 7, not 4, now value rounded up to 8